unstructured/metrics/table/table_formats.py

```diff
@@ -0,0 +1,38 @@
+from dataclasses import dataclass
+
+from more_itertools import first
```

Comment:
this is not currently in the dependencies and I would suggest just write the helper ourselves. Adding a dependency to an already complex open source project tend to create unintended dependency issues down the road.